notebooks/125-torchvision-zoo-to-openvino/125-convnext-classification.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<p>it will be also great if you show how to see all available models and add some words about parameters used for model initialization</p><p><a href="https://pytorch.org/vision/stable/models.html" target="_blank">https://pytorch.org/vision/stable/models.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
added